### ISSUE TITLE
Query regarding date of birth if born after midnight

### ISSUE THREAD

**Issue by rajagrawal1** [VedAstro/VedAstro#117]
I'm an absolute beginner in learning vedic astrology, and I learnt that date changes only after sunrise. Taking that into consideration and VedAstro, should I consider inputting the date `before` `or after 12am` `if one is born at 1am`? I'm still looking in the codebase to find this information and thought of asking instead.

Dhanyawad for for creating this gem of a project.

**Comment by sengiv** [comment]
Yes the sunrise & sunset system for a day in vedic astrology is fully implemented in VedAstro.
BUT
this is not the end user's concern, input the exact time as it appeared on the hospital clock at birth time, everything else is auto handled from timezone correction to LMT conversion

use JHora to verify with VedAstro

**Comment by rajagrawal1** [comment]
@sengiv Thank you for the clarification. If I understood right, if someone is born on 2nd of January, 1am as per western astrology, the Vedic equivalent would be 1st of January 1am. But the end user should still input 2nd January, 1am since the logic is automagically handled. Is this my correct understanding?

**Comment by sengiv** [comment]
Yes, correct
